### PR TITLE
Milestone Components and Redux Store

### PR DESCRIPTION
**Pull request by SantiagoGuerra**
Hello kind reviewer, in the following list you'll find more information about the work done in this PR:

- Implemented components
- Add Store
- Use Redux State in the logic
- Initialize the state with some book
- Each book have its property, including the author property

Thanks in advance!

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
```
Comment (shloch) [inline src/components/BookList.js]: this component should be in the folder : ` src/containers`
Screen Shot 2020-07-14 at 12 03 19 AM

File: src/components/BookForm.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BookForm.js]: Screen Shot 2020-07-14 at 12 03 19 AM

should be in folder `/sc/containers`

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @SantiagoGuerra 
We're good to go

Small change below, that has minor effect on your code, but just to keep up with the requirements
Please do this small change
Thanks

**Review by shloch — APPROVED** [review]
accomplished
good to go

**Comment by SantiagoGuerra** [comment]
Hi @shloch , apparently you forgot to change the status, hehe

Screenshot from 2020-07-14 20-44-58

I'd thank you so much to help me with this, and thank you in advance again!

**Comment by shloch** [comment]
Hello @SantiagoGuerra  .....Am so sorry, must have just forgotten.....please just continue with the next milestone.....I will contact the backend team to validate manually .........

**Comment by shloch** [comment]
Hello @SantiagoGuerra , you can check now, i have a confirmation that it is validated .....just hit me up on slack for confirmation
Good day

**Comment by SantiagoGuerra** [comment]
@shloch I could not find you on Slack, but thank you so much for the help, the issue is solved :)

**Comment by shloch** [comment]
on slack, i am <SHEY Louis CHIA>, any ways, good to know
Good day

**Pull request merged**